ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_order_statistics.py

```diff
@@ -6,10 +6,42 @@
 from ivy_tests.test_ivy.test_functional.test_core.test_statistical import (
     _statistical_dtype_values,
 )
+import ivy_tests.test_ivy.helpers as helpers
 import ivy_tests.test_ivy.test_frontends.test_numpy.helpers as np_frontend_helpers
 from ivy_tests.test_ivy.helpers import handle_frontend_test
 
 
+# ptp
+@handle_frontend_test(
+    fn_tree="numpy.ptp",
+    dtype_values_axis=_statistical_dtype_values(function="ptp"),
+    keep_dims=st.booleans(),

```

Comment:
Need to add the `test_without` value here to have
```suggestion
    keep_dims=st.booleans(),
    test_without=st.just(False),

```